### ISSUE TITLE
how do I get gifts from Winterfest?

### ISSUE THREAD

**Issue by prtolem** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#374]
### Issue Checklist

- [X] I have properly named the issue
- [X] I looked in issues/discussions, if my question was previously anwsered

### What is your question?

I found a method (https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/5863ceb55b6ea2dca9d2eece3da14e4e327ff45e/EpicGames/FN-Service/Game/Profile/Operations/UnlockRewardNode.md), but it requires some parameters such as NodeID and rewardGraphId. Where can I get them? Thanks!

**Comment by BurakYs** [comment]
In the game files:  FortniteGame/Plugins/GameFeatures/Winterfest/Content/Items/RewardGraph/S33/S33_Winterfest.uasset

**Comment by prtolem** [comment]
> In the game files: FortniteGame/Plugins/GameFeatures/Winterfest/Content/Items/RewardGraph/S33/S33_Winterfest.uasset

Thank you! Do you have information about the available gifts? not in kind .uasset, in plain-text

**Comment by BurakYs** [comment]
Yes athena QueryProfile has a `reward_nodes_claimed` prop in items, the templateId of the item is `AthenaRewardGraph:s33_winterfest`
The prop is added when the player opens at least one present though

**Comment by prtolem** [comment]
what if the user has not opened any gifts?

**Comment by LeleDerGrasshalmi** [comment]
Then that property does **not exist**:

```json
{
    "templateId": "AthenaRewardGraph:s33_winterfest",
    "attributes": {
        "unlock_epoch": "2024-12-19T14:00:00.000Z",
        "player_random_seed": 238309980,
        "reward_graph_purchased_timestamp": 1735143642029,
        "reward_graph_purchased": true,
        "reward_keys": [
            {
                "static_key_template_id": "Token:athena_s33_winterfest_key",
                "static_key_max_count": 14,
                "static_key_initial_count": 0,
                "unlock_keys_used": 0
            }
        ]
    },
    "quantity": 1
}
```

So simply fallback to an empty array

![image](https://github.com/user-attachments/assets/9ba97616-cc0d-4893-8d6c-8c7f7dc46eae)

**Comment by LeleDerGrasshalmi** [comment]
stale/resolved
